### DISCUSSION TITLE
(merged with #1914) Make alarm/ringer vibrations as long as on the smartphone instead of only brief jolts, and show the weather on all watchfaces

### DISCUSSION THREAD

**Discussion by smaybius** [InfiniTimeOrg/InfiniTime#1898] [Ideas]
So far, the vibration of both the ringer and the alarm is only brief jolts instead of full-on rumbles that are as long as the phone's. The weather should also be displayed on all watchfaces instead of only the PineTimeStyle, and the user should be able to switch it between C and F.

**Comment by jmlich** [comment]
Notifications Controller implementation currently doesn't provide way to dismiss notification via bluetooth. Its definitely interesting feature.  

If I understand correctly, weather service is still in proof of concept phase:
- https://github.com/InfiniTimeOrg/InfiniTime/pull/1890 - limit on number of items (keep memory in some bounds)
- weather app is disabled by default
- there is pull requests to add weather to digital watch face https://github.com/InfiniTimeOrg/InfiniTime/pull/1856
- also temperature unit is already reported https://github.com/InfiniTimeOrg/InfiniTime/issues/1783

**Reply by FintasticMan** [reply]
The weather service is fully implemented, supported by companions and displays on the PTS watch face. The issues are that the current implementation doesn't allow for all the functionality that a user might expect. I'm quite sure that no memory is being leaked, there is just no upper limit to the amount of memory. My PR addresses these issues.

**Reply by jmlich** [reply]
Oh I see, its not about leak,but it stores all data companion app sends. I remembered only that it ended up by reboot because out of memory.  I am sorry for misinformation. But still, its looks like incomplete feature, because the application is intentionally disabled and other watch faces doesn't support it at all.

**Reply by FintasticMan** [reply]
The app that's disabled was only ever a debug app, we have just haven't put in the effort make an app with a user-friendly UI.

**Reply by jmlich** [reply]
Yes, I am not native in English, so I was probably inaccurate. I hope that the memory limit being merge until next release. I can probably take a look into improvement of debug app.
